fwoa-tools/src/binaryConverter.ts

```diff
@@ -158,17 +158,16 @@ export async function convertBinaryResource(outputFile: ExportOutput): Promise<v
       await uploadBinaryResource(newKey, results, s3Client);
       logs.write(`${new Date().toISOString()}: Updated Binary .ndjson uploaded to Export Bucket!\n`);
     }
-    results = results.trimEnd();
```

Comment:
previous CR had this on line ~140 is it needed?

Reply:
It isn't needed, we can just remove the '\n' and this line since we're only doing 1 resource per ndjson file.